static/monorail-2.0.yaml

```diff
@@ -3231,7 +3232,7 @@ paths:
           $ref: '#/definitions/post_tasks'
       responses:
         201:
-          description: Successfully posted the specified task
+          description: Successfully started a specific task
```

Comment:
This is not "start a task", `POST /task/:identifier` is used in microkernel to send back the command execution result to RackHD.